envs/production/api.tf

```diff
@@ -3,7 +3,7 @@ locals {
     image_tags = {
       database_layer   = "0.3.78"
       server           = "production"
-      api_db_migration = "0.18.0"
+      api_db_migration = "0.20.0"
```

Comment:
Does this contain another migration - apart from adding a description - that we do not want to include?

Reply:
The [migration 0.19.0](https://github.com/serlo/db-migrations/pull/327/files) would create an exercise_submission table. This does not seem dangerous to me but maybe give it a short look before deploying @hugotiburtino.